### PR TITLE
Removed double spaces from .cs and .ps1 files

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Did a regular expression search "\b  \b" in VSCode while checking if https://github.com/PowerShell/PowerShell/pull/4741/files missed any, so fixing the ones in .cs and .ps1 files.

A few left in that help align comments making them easier to read.

Removed one commented out code that doesn't provide value.

The file SessionStateDriveAPIs.cs has a section that I didn't change, but git complained it had LF line endings and resaving as CRLF, so maybe only that section had mixed line endings.

### CODE REVIEW

File: src/System.Management.Automation/engine/CoreAdapter.cs [hunk 1]
```diff
@@ -3122,7 +3122,7 @@ internal static bool DisallowPrivateReflection(TypeInfo typeInfo)
             }
 
             var productAttribute = assembly.GetCustomAttribute<AssemblyProductAttribute>();
-            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft� .NET Framework", StringComparison.OrdinalIgnoreCase))
+            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft� .NET Framework", StringComparison.OrdinalIgnoreCase))
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/CoreAdapter.cs]: Please revert the change.

File: src/System.Management.Automation/engine/SessionStateDriveAPIs.cs [hunk 2]
```diff
@@ -508,13 +508,13 @@ private PSDriveInfo GetDrive(string name, bool automount)
...
-                {
-                    result = AutomountBuiltInDrive(name); // internally this calls GetDrive(name, false)
-                }
+                // first try to automount as a file system drive
+                result = AutomountFileSystemDrive(name);
+                // if it didn't work, then try automounting as a BuiltIn drive (e.g. "Cert"/"Certificate"/"WSMan")
+                if (result == null)
+                {
+                    result = AutomountBuiltInDrive(name); // internally this calls GetDrive(name, false)
+                }
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/SessionStateDriveAPIs.cs]: I wonder what is changed here? EOL?

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs [hunk 3]
```diff
@@ -525,7 +525,7 @@ protected override void StopProcessing()
         private CustomSerialization _serializer;
 
         /// <summary>
-        ///Memory Stream used for  serialization
+        ///Memory Stream used for serialization
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs]: Please add space before `Memory`.
It seems we should set final point.

File: src/System.Management.Automation/engine/CoreAdapter.cs (hunk 1 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/CoreAdapter.cs]: Will revert this

File: src/System.Management.Automation/engine/SessionStateDriveAPIs.cs (hunk 2 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/SessionStateDriveAPIs.cs]: Git gave some warnings about line endings (and performing some LF<->CRLF conversions on my behalf since I was doing this on Windows), so I think that's what this is.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs (hunk 3 above)
Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs]: Ok

File: src/System.Management.Automation/engine/SessionStateDriveAPIs.cs (hunk 2 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/SessionStateDriveAPIs.cs]: Thanks for clarify. 
Closed.

**Review by iSazonov — APPROVED** [review]
So many files - we need an approvement from someone else.

File: src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatViewManager.cs [hunk 4]
```diff
@@ -95,7 +95,7 @@ private static string PSObjectTypeName(PSObject so)
                                     FormattingCommandLineParameters parameters)
         {
             ViewDefinition view = null;
-            const string findViewType = "FINDING VIEW  TYPE: {0}";
+            const string findViewType = "FINDING VIEW TYPE: {0}";
             const string findViewShapeType = "FINDING VIEW {0} TYPE: {1}";
             const string findViewNameType = "FINDING VIEW NAME: {0}  TYPE: {1}";
```
Comment (lzybkr) [inline src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatViewManager.cs]: You missed one.

File: src/System.Management.Automation/engine/CoreAdapter.cs [hunk 5]
```diff
@@ -3122,7 +3122,7 @@ internal static bool DisallowPrivateReflection(TypeInfo typeInfo)
             }
 
             var productAttribute = assembly.GetCustomAttribute<AssemblyProductAttribute>();
-            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft� .NET Framework", StringComparison.OrdinalIgnoreCase))
+            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft® .NET Framework", StringComparison.OrdinalIgnoreCase))
```
Comment (lzybkr) [inline src/System.Management.Automation/engine/CoreAdapter.cs]: Can we change this to `Microsoft\u00a9` (assuming that is the correct character). When we run automated tools, they always mess this character up.

File: src/System.Management.Automation/engine/SessionStateDriveAPIs.cs (hunk 2 above)
Comment (lzybkr) [reply src/System.Management.Automation/engine/SessionStateDriveAPIs.cs]: The original file had inconsistent line endings, `LF` everywhere but this region. Your script changed the file to have consistent line endings, so git sees this region differently I guess.

File: src/System.Management.Automation/engine/CoreAdapter.cs (hunk 5 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/CoreAdapter.cs]: We have the character in some files - maybe fix this all in follow PR?

File: src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatViewManager.cs (hunk 4 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/commands/utility/FormatAndOutput/common/FormatViewManager.cs]: will fix

File: src/System.Management.Automation/engine/CoreAdapter.cs (hunk 5 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/CoreAdapter.cs]: Created issue https://github.com/PowerShell/PowerShell/issues/4752 as we should just fix that (and potentially Trademark as well)

**Review by lzybkr — APPROVED** [review]
